libs/agency/feature/dashboard/src/lib/components/notes/note-form/note-form.component.html

```diff
@@ -4,27 +4,32 @@
     <nuverial-icon class="notes-header-icon" *ngIf="this.mode === formModeEnum.Add" iconName="menu_open"></nuverial-icon>
   </header>
   <section class="notes-form">
+    <header *ngIf="formErrors.length > 0"><nuverial-form-errors [formErrors]="formErrors"></nuverial-form-errors></header>
```

Comment:
This should be sufficient

```suggestion
    <header *ngIf="formErrors.length"><nuverial-form-errors [formErrors]="formErrors"></nuverial-form-errors></header>
```